src/foundry/client-esm/data/region-behaviors/adjust-darkness-level.d.mts

```diff
@@ -1,29 +1,37 @@
 import type RegionBehaviorType from "./base.d.mts";
-import type { ValueOf } from "../../../../utils/index.d.mts";
+import type { Brand } from "../../../../utils/index.d.mts";
 import fields = foundry.data.fields;
+import type { InvertObject } from "../../../common/utils/helpers.d.mts";
 
 declare namespace AdjustDarknessLevelRegionBehaviorType {
-  const _MODES: Readonly<{
-    /**
-     * Override the darkness level with the modifier.
-     */
-    OVERRIDE: 0,
-
-    /**
-     * Brighten the darkness level: `darknessLevel * (1 - modifier)`
-     */
-    BRIGHTEN: 1,
-
-    /**
-     * Darken the darkness level: `1 - (1 - darknessLevel) * (1 - modifier)`.
-     */
-    DARKEN: 2
-  }>
-  type MODES = ValueOf<typeof _MODES>;
+  type _MODES = Brand<number, "AdjustDarknessLevelRegionBehaviorType.Modes">;
```

Comment:
Can you please name this `MODES` not `_MODES`. The underscore makes it seem like it should be internal but a valid has a valid reason to access this type.